gulpfile.js

```diff
@@ -15,16 +16,29 @@ var lintPaths = [
     'gulpfile.js'
 ];
 
-gulp.task('test', function(done) {
+gulp.task('media:watch', function () {
+    return gulp.src('./media/**/*')
+        .pipe(watch('./media/**/*', {
```

Comment:
Not currently. The .less files are still being compiled by django-pipeline. We've only disabled the asset collecting. We can do the less compilation here too, if you can find a way to turn it off? either way, this is still a considerable time-saver as-is.